Tests/LibWeb/Text/expected/wpt-import/webstorage/event_no_duplicates.txt

```diff
@@ -0,0 +1,13 @@
+Harness status: OK
+
+Found 8 tests
+
+8 Pass
+Pass	sessionStorage exists
+Pass	Setting to same value does not trigger event for sessionStorage
+Pass	Deleting non-existent key does not trigger event for sessionStorage
+Pass	Clearing empty storage does not trigger event for sessionStorage
+Pass	localStorage exists
+Pass	Setting to same value does not trigger event for localStorage
+Pass	Deleting non-existent key does not trigger event for localStorage
+Pass	Clearing empty storage does not trigger event for localStorage
```

Comment:
Ah, dang, of course - this specific subtest is inherently flakey if any other test is run in parallel with same backing storage. hmm, maybe could locally change this specific test to skip the localStorage case in particular